scripts/issue/367/dogfood_oxd.py

```diff
@@ -0,0 +1,36 @@
+from brownie import interface
+from helpers.addresses import registry
+from great_ape_safe import GreatApeSafe
+
+
+def main():
+    '''
+    dogfood half oxd into bveoxd,
+    lp into bveoxd/oxd,
+    dogfood lp into bbveoxd/oxd
+    '''
+    trops = GreatApeSafe(registry.ftm.badger_wallets.treasury_ops_multisig)
+
+    oxd = interface.ERC20(registry.ftm.treasury_tokens.OXD, owner=trops.account)
+    bveoxd = interface.ISettV4h(registry.ftm.sett_vaults.bveOXD, owner=trops.account)
+    bveoxd_oxd = interface.IUniswapV2Pair(registry.ftm.lp_tokens['bveOXD-OXD'], owner=trops.account)
+    bbveoxd_oxd = interface.ISettV4h(registry.ftm.sett_vaults['bbveOXD-OXD'], owner=trops.account)
+
+    trops.init_solidly()
+    trops.take_snapshot(tokens=[oxd, bveoxd, bbveoxd_oxd])
+
+    half_oxd = oxd.balanceOf(trops) / 2
+    oxd.approve(bveoxd, half_oxd)
+    bveoxd.deposit(half_oxd)
+
+    oxd_bal = oxd.balanceOf(trops)
+    bveoxd_bal = bveoxd.balanceOf(trops)
+    trops.solidly.add_liquidity(bveoxd, oxd, bveoxd_bal, oxd_bal)
```

Comment:
```suggestion
    trops.solidly.add_liquidity(bveoxd, oxd, bveoxd_bal * .99, oxd_bal)
```
i believe this needs to be dusty as well, in case ppfs increases between posting and execing.

Reply:
ah ya thats right, oxd is not 1:1 with bveoxd unlike bvecvx